pkg/api/v1/clients/kube/cache/cache_test.go

```diff
@@ -166,3 +168,14 @@ var _ = Describe("kube core cache tests", func() {
 func init() {
 	rand.Seed(time.Now().UnixNano())
 }
+
+// remove the auto-generated kube-root-ca.crt ConfigMap, if it exists
+func cleanConfigMaps(configMaps []*v1.ConfigMap) []*v1.ConfigMap {
```

Comment:
Do we know why this suddenly started getting generated?

Reply:
unfortunately not. from looking at the code, it seems like it should have always been generated, but not sure why this only started popping up yesterday